docs/2018-10-24-Configuration-Splitting-Simplification.md

```diff
@@ -0,0 +1,88 @@
+Configuration Splitting Simplification
+======================================
+
+Background
+----------
+
+When doing one-path reachability proofs and applying an axiom,
+`∀ Z . α(Z) → ● β(Z)`, where `●` is the strong-next symbol (aka 'next'),
+to the following reachability goal
+```
+∀ X . φ(X) → ◆ ∃ Y . ψ(X, Y)
+```
+the result will look something like
+```
+￼(∀ X . Φ'(X) → ◆ ∃ Y . ψ(X, Y))
+￼∧
+￼(∀ X . Φ(X) ∧ (¬ ∃ Z. α(Z)) → ● ◆ ∃ Y . ψ(X, Y))
```

Comment:
I agree that it's odd, I made similar comments about Traian's PR (both in writing and in-person). I think he managed to prove this, but it's still odd. Since it is copied from his doc, I'll add a TODO to update this when he merges.